birdhouse/components/README.rst

```diff
@@ -59,6 +59,12 @@ component directory to the ``BIRDHOUSE_EXTRA_CONF_DIRS`` variable in your local
   
   * component location: ``optional-components/scheduler-job-deploy_raven_testdata``
 
+* Automatically backup files
+
+  * automatically backs up application data, user data, and logs to a restic repository (default) or a docker volume
```

Comment:
Problably should have a cross-reference to the README section where `restic` details and `backup` feature should be further described (see other comments).